google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractReadContext.java

```diff
@@ -375,9 +380,15 @@ void initTransaction() {
           }
           transactionId = transaction.getId();
           span.addAnnotation(
-              "Transaction Creation Done", TraceUtil.getTransactionAnnotations(transaction));
+              "Transaction Creation Done",
+              ImmutableMap.of(
+                  "id",
```

Comment:
nit: is it intentional that this is `id` and the other is `Timestamp` (i.e. all lower case vs. starting with upper case)?